### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by Busiwa24**
In the following pull request
-I add an HTML file from the Microverse trials
-I add a CSS file
- run a linter check
- review the project that we attempted

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  <nav id="navbar">
+
+    <header>JS Documentation</header>
+    <ul>
+       <li><a class="nav-link" href="#Introduction">Introduction</a></li>
+       <li><a class="nav-link" href="#What_you_should_already_know">What you should already know</a></li>
+       <li><a class="nav-link" href="#JavaScript_and_Java">Javascript and Java</a></li>
+       <li><a class="nav-link" href="#Hello_world">Hello World</a></li>
+       <li><a class="nav-link" href="#Variables">Variables</a></li>
+    </ul>
```
Comment (Busiwa24) [inline index.html:20]: Nice job implementing top navigation. You did well using the header tag for the navigation. It is also good practice to wrap your header tags in a nav tag as this helps with making your code more readable to other users

For this part of the code surround your UL with the nav tag👌

File: index.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+   <section class="main-section" id="Hello_world">
+     <h1>Hello world</h1>
+     <p>
+       To get started with writing JavaScript,
+       open the Scratchpad and write your first "Hello world" JavaScript code:
+     </p>
+     <code>function greetMe(yourName) { alert("Hello " + yourName); }</code>
+     <p>
+       Select the code in the pad and hit Ctrl+R to watch it unfold in your browser!
+     </p>
```
Comment (Busiwa24) [inline index.html:118]: For this section, you could use the right semantic tag .eg span tag to display text spanning over a line. Using a <p> tag for a single line of code is not the best practice as the browser interprets this tag as containing a block of text as it is in an actual book

File: style.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+* {
```
Comment (Busiwa24) [inline style.css:1]: You could have added more styling to improve the User experience, eg it is good practice to use the same styling for the same tags such as the h1 tags in your code.

**Review by Busiwa24 — COMMENTED** [review]
Great so far, almost there.😎
You need to make a few changes to finish this project

**Review by Temmarie — APPROVED** [review]
## Status: Approved ✔️ 
### Hi @Busiwa24  , 
#### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
#### Congratulations! 🎉 

#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**